.ci/containers/membership-checker/main.go

```diff
@@ -91,12 +91,13 @@ func main() {
 			}
 		} else {
 			addAwaitingApprovalLabel(prNumber, GITHUB_TOKEN)
+			postAwaitingApprovalBuildLink(prNumber, GITHUB_TOKEN, projectId, commitSha)
 		}
 	}
 
 	// in community-checker job:
 	// remove awaiting-approval label from external contributor PRs
-	if target == "needs_approval" && !trusted {
+	if target == "needs_approval" {
```

Comment:
This change is to prevent labels not being removed for PRs whose contributor joins the org after creating the PR (e.g. https://github.com/GoogleCloudPlatform/magic-modules/pull/7610

Reply:
Does `removeAwaitingApprovalLabel` remove the label only if present?

I worry if it tries to remove something that doesn't exist the call to github will error out.. Although looking into the call it seems just try to remove regardless of presence and the api tolerates it. Is this so?

Reply:
It will return a 404 status code for non-exist labels, but won't error out. So I think it should be fine.